atom/browser/api/atom_api_desktop_capturer.cc

```diff
@@ -135,7 +135,10 @@ bool DesktopCapturer::ShouldScheduleNextRefresh(DesktopMediaList* list) {
   return false;
 }
 
-void DesktopCapturer::UpdateSourcesList(DesktopMediaList* list) {
+void DesktopCapturer::UpdateSourcesList(mate::Arguments* args) {
+  DesktopMediaList* list;
+  args->GetNext(&list);
```

Comment:
This isn't guaranteed to succeed, in order to offload the argument parsing to native mate we can do this instead.

```
void DesktopCapturer::UpdateSourcesList(DesktopMediaList* list, mate::Arguments* args) {
```